src/cert_utils.erl

```diff
@@ -0,0 +1,86 @@
+-module(cert_utils).
```

Comment:
Module description comment

Reply:
E.g. what certificates, what is the primary user of this module...